addons/project_sms/views/project_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_task_act_window_sms_composer" model="ir.actions.act_window">
+        <field name="name">Send SMS Text Message</field>
+        <field name="res_model">sms.composer</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+        <field name="context">{
+            'default_composition_mode': 'mass',
+            'default_mass_keep_log': True,
+            'default_res_ids': active_ids,
+        }</field>
+        <field name="binding_model_id" ref="model_project_task"/>
+        <field name="binding_view_types">list</field>
+    </record>
+
+    <record id="project_project_act_window_sms_composer" model="ir.actions.act_window">
+        <field name="name">Send SMS Text Message</field>
+        <field name="res_model">sms.composer</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+        <field name="context">{
+            'default_composition_mode': 'mass',
+            'default_mass_keep_log': True,
+            'default_res_ids': active_ids,
+        }</field>
+        <field name="binding_model_id" ref="model_project_project"/>
+        <field name="binding_view_types">list</field>
+    </record>
```

Comment:
Where are those actions used? 🤔

Reply:
Moreover, those actions do not open project records so the file should be renamed to `sms_composer_views.xml`. Also, this file should be moved in the `wizard` folder since `sms.composer` is a transient model (a wizard).

Reply:
@xavierbol , Same as the above comment, but for SMS. Thanks

Reply:
Indeed, my bad. 😅